index/login.php

```diff
@@ -77,13 +77,16 @@
 include("../includes/tjek4opdat.php");
 include("../includes/std_func.php");
 
-print "<!--";
-$timezone = system("timedatectl | grep \"Time zone\"");
-print "-->";
-list($tmp,$timezone) = explode(":",$timezone);
-list($timezone,$tmp) = explode("(",$timezone);
-$timezone = trim($timezone);
-if (!$timezone) $timezone = 'Europe/Copenhagen';
+#print "<!--";
+$timezone = system("timedatectl 2>/dev/null | grep \"Time zone\"", $errcode);
```

Comment:
$errorcode not defined before usage in system function

Reply:
Yes, `$errcode` is assigned by reference internally. Therefore initialization is not needed.

Try running this code in a terminal:
```
<?php
$good_command = system("ls >/dev/null", $errcode1);
var_dump($errcode1);  // -> int(0) as the ls command was succesful

$bad_command = system("invalid_command >/dev/null", $errcode2);
var_dump($errcode2); // -> int(127) as the command invalid_command was not found
?>
```